### PR TITLE
Update API docs

### PR DESCRIPTION
**Pull request by flaurida**
Note: some links in main docs will likely be broken since we changed our API, but these will be handled in a separate PR.

Part of #468

### CODE REVIEW

File: docs/docs/create_a_test.md [hunk 1]
```diff
@@ -65,27 +65,27 @@ Our code first requires the `qawolf` library, which is built on top of [Microsof
 The test code opens a browser and navigates to the specified URL. Each action you took in the browser is captured in a [Jest test](https://jestjs.io/docs/en/api#testname-fn-timeout). After the test finishes running, the browser is closed. Below we show our test code:
 
 ```js
-const { launch } = require("qawolf");
-const selectors = require("../selectors/myFirstTest");
+const { launch } = require('qawolf');
```
Comment (jperl) [inline docs/docs/create_a_test.md]: This is totally incorrect

Comment (flaurida) [reply docs/docs/create_a_test.md]: I am going to do the main docs separately, probably over a few PRs

**Comment by jperl** [comment]
Also please add missing `qawolf.waitForPage`

File: docs/docs/api/qawolf/register.mdx [hunk 2]
```diff
@@ -10,7 +10,10 @@ import Syntax from '../components/Syntax';
 
 <Syntax code="register(context)" />
 
-Allow QA Wolf to access the specified [Playwright `BrowserContext`](https://github.com/microsoft/playwright/blob/master/docs/api.md#class-browsercontext) instance. This enables the [default selector logic](../../use_custom_selectors#default-selector-logic) to work. It also automatically creates [debug artifacts](../../run_tests_in_ci#debug) for your tests and scripts.
+
```
Comment (flaurida) [inline docs/docs/api/qawolf/register.mdx]: Please add a high level summary statement. Then these can come below it, probably in bulleted list

File: docs/docs/api/qawolf/register.mdx [hunk 3]
```diff
@@ -10,7 +10,10 @@ import Syntax from '../components/Syntax';
...
+- Register the [Playwright `BrowserContext`](https://github.com/microsoft/playwright/blob/master/docs/api.md#class-browsercontext) so [waitForPage](TODO) works.
```
Comment (flaurida) [inline docs/docs/api/qawolf/register.mdx]: what is `waitForPage`? if part of API needs to be documented no?

File: docs/docs/api/qawolf/register.mdx (hunk 2 above)
Comment (jperl) [reply docs/docs/api/qawolf/register.mdx]: I outlined the bullets, feel free to add the summary.

File: docs/docs/api/qawolf/register.mdx (hunk 3 above)
Comment (jperl) [reply docs/docs/api/qawolf/register.mdx]: Yes see https://github.com/qawolf/playwright-utils/blob/master/src/context/waitForPage.ts

File: docs/docs/api/components/qawolf/ArgumentLaunch.js [hunk 4]
```diff
@@ -21,7 +21,10 @@ function ArgumentLaunch() {
       <Argument
         description={
           <React.Fragment>
-            Launch this type of browser. <b>Default:</b> <code>"chromium"</code>
+            Launch this type of browser. <b>Default:</b>{' '}
```
Comment (flaurida) [inline docs/docs/api/components/qawolf/ArgumentLaunch.js]: did you check this link works? typically you need to use `Link` and `useBaseUrl` for links within docs

Comment (jperl) [reply docs/docs/api/components/qawolf/ArgumentLaunch.js]: I thought it would work after deploying. My B

Comment (flaurida) [reply docs/docs/api/components/qawolf/ArgumentLaunch.js]: nope can you fix? search for `useBaseUrl` for an example

File: docs/docs/api/components/qawolf/ArgumentWaitForPage.js [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+  return (
+    <React.Fragment>
+      <Argument
+        description={
+          <React.Fragment>
+            The{' '}
+            <a href="https://github.com/microsoft/playwright/blob/master/docs/api.md#class-browsercontext">
+              Playwright <code>BrowserContext</code>
+            </a>{' '}
+            instance where the page will load.
```
Comment (jperl) [inline docs/docs/api/components/qawolf/ArgumentWaitForPage.js]: to load the page from

File: docs/docs/api/components/qawolf/ArgumentWaitForPage.js [hunk 6]
```diff
@@ -0,0 +1,63 @@
...
+          </React.Fragment>
+        }
+        name="context"
+        type="BrowserContext"
+      />
+      <Argument
+        description={
+          <React.Fragment>
+            The index of the page, starting from <code>0</code>.
```
Comment (jperl) [inline docs/docs/api/components/qawolf/ArgumentWaitForPage.js]: at

File: docs/docs/api/qawolf/register.mdx [hunk 7]
```diff
@@ -10,10 +10,11 @@ import Syntax from '../components/Syntax';
 
 <Syntax code="register(context)" />
 
+Allow QA Wolf to access the specified [Playwright `BrowserContext`](https://github.com/microsoft/playwright/blob/master/docs/api.md#class-browsercontext) instance, which:
```
Comment (jperl) [inline docs/docs/api/qawolf/register.mdx]: Register the browser context with QA Wolf which:

File: docs/docs/api/qawolf/wait_for_page.mdx [hunk 8]
```diff
@@ -0,0 +1,49 @@
...
+title: qawolf.waitForPage
+---
+
+import ArgumentWaitForPage from '../components/qawolf/ArgumentWaitForPage';
+import ReturnPage from '../components/qawolf/ReturnPage';
+import Syntax from '../components/Syntax';
+
+<Syntax code="waitForPage(context, index[, options])" />
+
+Wait for [Playwright `Page`](https://github.com/microsoft/playwright/blob/master/docs/api.md#class-page) instance with specified index to load.
```
Comment (jperl) [inline docs/docs/api/qawolf/wait_for_page.mdx]: Wait for a page based on it’s creation index.

**Pull request merged**